README.md

```diff
@@ -448,6 +448,34 @@ The following permissions are required for the preceding configuration of "repro
 }
 ```
 
+### How to deploy faster with Auto Scaling Group
+
+Add following configuration to your deploy.rb and hooks if you need.
+
+```ruby
+# deploy.rb
+set :ecs_instance_fluctuation_manager_configs, [
+  {
+    region: "ap-northeast-1",
+    cluster: "CLUSTER_NAME",
+    cluster_to_asg: { "CLUSTER_NAME" => "AUTO_SCALING_GROUP_NAME" },
```

Comment:
Because I transplanted frankly. Now, I think this is redundant code, so we can remove `cluster_to_asg` and `cluster` and add `auto_scaling_group` instead.

Reply:
Fixed in 65a06e4